server/src/main/java/com/hedera/block/server/persistence/storage/read/BlockAsDirReader.java

```diff
@@ -128,8 +133,10 @@ public Optional<Block> read(final long blockNumber) throws IOException {
     private Optional<BlockItem> readBlockItem(@NonNull final String blockItemPath)
             throws IOException {
 
-        try (FileInputStream fis = new FileInputStream(blockItemPath)) {
-            return Optional.of(BlockItem.parseFrom(fis));
+        try (@NonNull final FileInputStream fis = new FileInputStream(blockItemPath)) {
```

Comment:
`@NonNull` is unnecessary here, as `new` cannot return null.

Reply:
Fixed